src/main/java/com/ays/backend/user/controller/payload/request/RegisterRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.ays.backend.user.controller.payload.request;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Data
+@Builder
+public class RegisterRequest {
+
+    private String username;
+    private String password;
+    private String countryCode;
+    private String lineNumber;
+    private String firstName;
+    private String lastName;
+    private String email;
+    private Long organizationId;
```

Comment:
I think validations have been forgotten, let's make sure to add validations for each field.

Reply:
I just added validations into RegisterRequest